[32mINFO    [0m juju.model:model.py:2097 Deploying local:opensearch-0
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:329 



[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:330 Model  Controller           Cloud/Region         Version  SLA          Timestamp
test   localhost-localhost  localhost/localhost  3.5.3    unsupported  01:46:23Z

App                       Version  Status   Scale  Charm                     Channel        Rev  Exposed  Message
opensearch                         active       2  opensearch                                 0  no       
second-opensearch                  waiting    0/1  opensearch                                 0  no       waiting for machine
self-signed-certificates           active       1  self-signed-certificates  latest/stable  155  no       

Unit                         Workload  Agent       Machine  Public address  Ports     Message
opensearch/0*                active    idle        1        10.249.120.88   9200/tcp  
opensearch/1                 active    idle        2        10.249.120.92   9200/tcp  
second-opensearch/0          waiting   allocating  3                                  waiting for machine
self-signed-certificates/0*  active    executing   0        10.249.120.236            

Machine  State    Address         Inst id        Base          AZ  Message
0        started  10.249.120.236  juju-0cd968-0  ubuntu@22.04      Running
1        started  10.249.120.88   juju-0cd968-1  ubuntu@22.04      Running
2        started  10.249.120.92   juju-0cd968-2  ubuntu@22.04      Running
3        pending                  pending        ubuntu@22.04      Creating container

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


01:46:23 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:277 	App: opensearch - Units - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:278 
	app: opensearch active -- message: None
		opensearch-0.e92* -- (10.249.120.88) -- [idle (since: 01:44:53)] active: 
		opensearch-1.e92  -- (10.249.120.92) -- [idle (since: 01:46:11)] active: 

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


01:46:34 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:277 	App: opensearch - Units - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:278 
	app: opensearch active -- message: None
		opensearch-0.e92* -- (10.249.120.88) -- [idle (since: 01:44:53)] active: 
		opensearch-1.e92  -- (10.249.120.92) -- [idle (since: 01:46:11)] active: 

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


01:46:46 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:277 	App: opensearch - Units - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:278 
	app: opensearch active -- message: None
		opensearch-0.e92* -- (10.249.120.88) -- [idle (since: 01:44:53)] active: 
		opensearch-1.e92  -- (10.249.120.92) -- [idle (since: 01:46:11)] active: 

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


01:46:57 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:277 	App: opensearch - Units - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:278 
	app: opensearch active -- message: None
		opensearch-0.e92* -- (10.249.120.88) -- [idle (since: 01:44:53)] active: 
		opensearch-1.e92  -- (10.249.120.92) -- [idle (since: 01:46:11)] active: 

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


01:47:08 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:277 	App: opensearch - Units - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:278 
	app: opensearch active -- message: None
		opensearch-0.e92* -- (10.249.120.88) -- [idle (since: 01:44:53)] active: 
		opensearch-1.e92  -- (10.249.120.92) -- [idle (since: 01:46:11)] active: 

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


01:47:19 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:277 	App: opensearch - Units - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:278 
	app: opensearch active -- message: None
		opensearch-0.e92* -- (10.249.120.88) -- [idle (since: 01:44:53)] active: 
		opensearch-1.e92  -- (10.249.120.92) -- [idle (since: 01:46:11)] active: 

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


01:47:30 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:261 	App: second-opensearch - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:262 
	app: second waiting -- message: agent initialising
		second-opensearch-0.f50* -- (10.249.120.232) -- [allocating (since: 01:46:23)] waiting: agent initialising

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


01:47:42 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:261 	App: second-opensearch - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:262 
	app: second maintenance -- message: Installing OpenSearch...
		second-opensearch-0.f50* -- (10.249.120.232) -- [executing (since: 01:47:35)] maintenance: Installing OpenSearch...

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


01:47:54 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:261 	App: second-opensearch - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:262 
	app: second maintenance -- message: Installing OpenSearch...
		second-opensearch-0.f50* -- (10.249.120.232) -- [executing (since: 01:47:35)] maintenance: Installing OpenSearch...

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


01:48:06 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:261 	App: second-opensearch - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:262 
	app: second maintenance -- message: Installing OpenSearch...
		second-opensearch-0.f50* -- (10.249.120.232) -- [executing (since: 01:47:35)] maintenance: Installing OpenSearch...

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


01:48:18 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:261 	App: second-opensearch - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:262 
	app: second maintenance -- message: Installing OpenSearch...
		second-opensearch-0.f50* -- (10.249.120.232) -- [executing (since: 01:47:35)] maintenance: Installing OpenSearch...

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


01:48:29 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:261 	App: second-opensearch - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:262 
	app: second maintenance -- message: Installing OpenSearch...
		second-opensearch-0.f50* -- (10.249.120.232) -- [executing (since: 01:47:35)] maintenance: Installing OpenSearch...

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


01:48:41 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:261 	App: second-opensearch - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:262 
	app: second maintenance -- message: Waiting for TLS to be fully configured...
		second-opensearch-0.f50* -- (10.249.120.232) -- [executing (since: 01:48:42)] maintenance: Waiting for TLS to be fully configured...

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


01:48:53 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:261 	App: second-opensearch - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:262 
	app: second maintenance -- message: Waiting for TLS to be fully configured...
		second-opensearch-0.f50* -- (10.249.120.232) -- [executing (since: 01:48:54)] maintenance: Waiting for TLS to be fully configured...

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


01:49:05 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:261 	App: second-opensearch - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:262 
	app: second waiting -- message: Waiting for OpenSearch to start...
		second-opensearch-0.f50* -- (10.249.120.232) -- [executing (since: 01:48:56)] waiting: Waiting for OpenSearch to start...

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


01:49:17 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:277 	App: second-opensearch - Units - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:278 
	app: second active -- message: None
		second-opensearch-0.f50* -- (10.249.120.232) -- [executing (since: 01:49:18)] active: 

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


01:49:29 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:277 	App: second-opensearch - Units - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:278 
	app: second active -- message: None
		second-opensearch-0.f50* -- (10.249.120.232) -- [idle (since: 01:49:21)] active: 

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


01:49:40 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:277 	App: second-opensearch - Units - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:278 
	app: second active -- message: None
		second-opensearch-0.f50* -- (10.249.120.232) -- [idle (since: 01:49:21)] active: 

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


01:49:52 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:277 	App: second-opensearch - Units - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:278 
	app: second active -- message: None
		second-opensearch-0.f50* -- (10.249.120.232) -- [idle (since: 01:49:21)] active: 

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


01:50:04 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:277 	App: second-opensearch - Units - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:278 
	app: second active -- message: None
		second-opensearch-0.f50* -- (10.249.120.232) -- [idle (since: 01:49:21)] active: 

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


01:50:16 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:277 	App: second-opensearch - Units - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:278 
	app: second active -- message: None
		second-opensearch-0.f50* -- (10.249.120.232) -- [idle (since: 01:49:21)] active: 

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


01:50:27 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:277 	App: second-opensearch - Units - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:278 
	app: second active -- message: None
		second-opensearch-0.f50* -- (10.249.120.232) -- [idle (since: 01:49:21)] active: 

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


01:50:39 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:348 01:50:41 -- Waiting for model: complete.



[32mINFO    [0m integration.helpers:helpers.py:347 Calling: PUT -- https://10.249.120.88:9200/test_index_unique_cluster_dbs/_doc/1
[32mINFO    [0m integration.helpers:helpers.py:347 Calling: POST -- https://10.249.120.88:9200/test_index_unique_cluster_dbs/_refresh
[32mINFO    [0m integration.helpers:helpers.py:347 Calling: PUT -- https://10.249.120.232:9200/test_index_unique_cluster_dbs/_doc/2
[32mINFO    [0m integration.helpers:helpers.py:347 Calling: POST -- https://10.249.120.232:9200/test_index_unique_cluster_dbs/_refresh
[32mINFO    [0m integration.helpers:helpers.py:347 Calling: GET -- https://10.249.120.88:9200/test_index_unique_cluster_dbs/_search
[32mINFO    [0m integration.helpers:helpers.py:347 Calling: GET -- https://10.249.120.232:9200/test_index_unique_cluster_dbs/_search
[32mINFO    [0m integration.helpers:helpers.py:347 Calling: DELETE -- https://10.249.120.88:9200/test_index_unique_cluster_dbs/
[32mINFO    [0m integration.ha.helpers:helpers.py:247 Continuous writes result: namespace(max_stored_id=379, count=380, last_expected_id=379)
[32mINFO    [0m integration.helpers:helpers.py:347 Calling: GET -- https://10.249.120.88:9200/series_index/_search_shards
[32mINFO    [0m integration.ha.conftest:conftest.py:39 



The writes have been cleared.




[32mINFO    [0m pytest_operator.plugin:plugin.py:862 Model status:

Model  Controller           Cloud/Region         Version  SLA          Timestamp
test   localhost-localhost  localhost/localhost  3.5.3    unsupported  01:51:18Z

App                       Version  Status  Scale  Charm                     Channel        Rev  Exposed  Message
opensearch                         active      2  opensearch                                 0  no       
self-signed-certificates           active      1  self-signed-certificates  latest/stable  155  no       

Unit                         Workload  Agent  Machine  Public address  Ports     Message
opensearch/0*                active    idle   1        10.249.120.88   9200/tcp  
opensearch/1                 active    idle   2        10.249.120.92   9200/tcp  
self-signed-certificates/0*  active    idle   0        10.249.120.236            

Machine  State    Address         Inst id        Base          AZ  Message
0        started  10.249.120.236  juju-0cd968-0  ubuntu@22.04      Running
1        started  10.249.120.88   juju-0cd968-1  ubuntu@22.04      Running
2        started  10.249.120.92   juju-0cd968-2  ubuntu@22.04      Running
3        stopped  10.249.120.232  juju-0cd968-3  ubuntu@22.04      Running

[32mINFO    [0m pytest_operator.plugin:plugin.py:868 Juju error logs:

machine-0: 01:42:01 ERROR juju.worker.dependency "lxd-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-0: 01:42:01 ERROR juju.worker.dependency "kvm-container-provisioner" manifold worker returned unexpected error: container types not yet available
unit-self-signed-certificates-0: 01:42:01 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
machine-1: 01:42:30 ERROR juju.worker.dependency "kvm-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-1: 01:42:30 ERROR juju.worker.dependency "lxd-container-provisioner" manifold worker returned unexpected error: container types not yet available
unit-opensearch-0: 01:42:30 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
machine-2: 01:42:36 ERROR juju.worker.dependency "kvm-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-2: 01:42:36 ERROR juju.worker.dependency "lxd-container-provisioner" manifold worker returned unexpected error: container types not yet available
unit-opensearch-1: 01:42:36 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
unit-opensearch-0: 01:43:35 ERROR unit.opensearch/0.juju-log [Errno 111] Connection refused
unit-opensearch-0: 01:43:36 ERROR unit.opensearch/0.juju-log opensearch-peers:0: [Errno 111] Connection refused
unit-opensearch-0: 01:43:36 ERROR unit.opensearch/0.juju-log opensearch-peers:0: [Errno 111] Connection refused
unit-opensearch-0: 01:43:36 ERROR unit.opensearch/0.juju-log opensearch-peers:0: [Errno 111] Connection refused
unit-opensearch-0: 01:43:36 ERROR unit.opensearch/0.juju-log opensearch-peers:0: [Errno 111] Connection refused
unit-opensearch-0: 01:43:36 ERROR unit.opensearch/0.juju-log upgrade-version-a:2: [Errno 111] Connection refused
unit-opensearch-0: 01:43:37 ERROR unit.opensearch/0.juju-log certificates:3: [Errno 111] Connection refused
unit-opensearch-0: 01:43:40 ERROR unit.opensearch/0.juju-log [Errno 111] Connection refused
unit-opensearch-0: 01:43:41 ERROR unit.opensearch/0.juju-log [Errno 111] Connection refused
unit-opensearch-0: 01:43:42 ERROR unit.opensearch/0.juju-log [Errno 111] Connection refused
unit-opensearch-0: 01:43:43 ERROR unit.opensearch/0.juju-log certificates:3: [Errno 111] Connection refused
unit-opensearch-0: 01:43:43 ERROR unit.opensearch/0.juju-log certificates:3: [Errno 111] Connection refused
unit-opensearch-0: 01:43:43 ERROR unit.opensearch/0.juju-log certificates:3: Cannot connect to the OpenSearch server...
unit-opensearch-0: 01:43:43 ERROR unit.opensearch/0.juju-log certificates:3: [Errno 111] Connection refused
unit-opensearch-0: 01:43:44 ERROR unit.opensearch/0.juju-log [Errno 111] Connection refused
unit-opensearch-0: 01:43:44 ERROR unit.opensearch/0.juju-log Cannot connect to the OpenSearch server...
unit-opensearch-0: 01:43:44 ERROR unit.opensearch/0.juju-log [Errno 111] Connection refused
unit-opensearch-0: 01:43:44 ERROR unit.opensearch/0.juju-log [Errno 111] Connection refused
unit-opensearch-0: 01:43:44 ERROR unit.opensearch/0.juju-log Cannot connect to the OpenSearch server...
unit-opensearch-0: 01:43:44 ERROR unit.opensearch/0.juju-log [Errno 111] Connection refused
unit-opensearch-0: 01:43:44 ERROR unit.opensearch/0.juju-log Cannot connect to the OpenSearch server...
unit-opensearch-0: 01:43:47 ERROR unit.opensearch/0.juju-log [Errno 111] Connection refused
unit-opensearch-0: 01:43:47 ERROR unit.opensearch/0.juju-log Cannot connect to the OpenSearch server...
unit-opensearch-0: 01:43:50 ERROR unit.opensearch/0.juju-log [Errno 111] Connection refused
unit-opensearch-0: 01:43:50 ERROR unit.opensearch/0.juju-log Cannot connect to the OpenSearch server...
unit-opensearch-0: 01:43:53 ERROR unit.opensearch/0.juju-log [Errno 111] Connection refused
unit-opensearch-0: 01:43:53 ERROR unit.opensearch/0.juju-log Cannot connect to the OpenSearch server...
unit-opensearch-1: 01:44:10 ERROR unit.opensearch/1.juju-log [Errno 111] Connection refused
unit-opensearch-1: 01:44:11 ERROR unit.opensearch/1.juju-log [Errno 111] Connection refused
unit-opensearch-1: 01:44:12 ERROR unit.opensearch/1.juju-log upgrade-version-a:2: [Errno 111] Connection refused
unit-opensearch-1: 01:44:13 ERROR unit.opensearch/1.juju-log upgrade-version-a:2: [Errno 111] Connection refused
unit-opensearch-1: 01:44:13 ERROR unit.opensearch/1.juju-log opensearch-peers:0: [Errno 111] Connection refused
unit-opensearch-1: 01:44:13 ERROR unit.opensearch/1.juju-log opensearch-peers:0: [Errno 111] Connection refused
unit-opensearch-1: 01:44:14 ERROR unit.opensearch/1.juju-log node-lock-fallback:1: [Errno 111] Connection refused
unit-opensearch-1: 01:44:15 ERROR unit.opensearch/1.juju-log upgrade-version-a:2: [Errno 111] Connection refused
unit-opensearch-1: 01:44:16 ERROR unit.opensearch/1.juju-log certificates:3: [Errno 111] Connection refused
unit-opensearch-1: 01:44:18 ERROR unit.opensearch/1.juju-log [Errno 111] Connection refused
unit-opensearch-1: 01:44:19 ERROR unit.opensearch/1.juju-log [Errno 111] Connection refused
unit-opensearch-1: 01:44:20 ERROR unit.opensearch/1.juju-log certificates:3: [Errno 111] Connection refused
unit-opensearch-1: 01:44:20 ERROR unit.opensearch/1.juju-log certificates:3: [Errno 111] Connection refused
unit-opensearch-1: 01:44:20 ERROR unit.opensearch/1.juju-log certificates:3: Cannot connect to the OpenSearch server...
unit-opensearch-1: 01:44:20 ERROR unit.opensearch/1.juju-log certificates:3: [Errno 111] Connection refused
unit-opensearch-1: 01:44:21 ERROR unit.opensearch/1.juju-log [Errno 111] Connection refused
unit-opensearch-1: 01:44:21 ERROR unit.opensearch/1.juju-log Cannot connect to the OpenSearch server...
unit-opensearch-1: 01:44:21 ERROR unit.opensearch/1.juju-log [Errno 111] Connection refused
unit-opensearch-1: 01:44:22 ERROR unit.opensearch/1.juju-log [Errno 111] Connection refused
unit-opensearch-1: 01:44:22 ERROR unit.opensearch/1.juju-log Cannot connect to the OpenSearch server...
unit-opensearch-1: 01:44:22 ERROR unit.opensearch/1.juju-log [Errno 111] Connection refused
unit-opensearch-1: 01:44:22 ERROR unit.opensearch/1.juju-log [Errno 111] Connection refused
unit-opensearch-1: 01:44:23 ERROR unit.opensearch/1.juju-log node-lock-fallback:1: [Errno 111] Connection refused
unit-opensearch-1: 01:44:23 ERROR unit.opensearch/1.juju-log node-lock-fallback:1: Cannot connect to the OpenSearch server...
unit-opensearch-1: 01:44:23 ERROR unit.opensearch/1.juju-log node-lock-fallback:1: [Errno 111] Connection refused
unit-opensearch-1: 01:44:23 ERROR unit.opensearch/1.juju-log node-lock-fallback:1: [Errno 111] Connection refused
unit-opensearch-1: 01:44:24 ERROR unit.opensearch/1.juju-log node-lock-fallback:1: [Errno 111] Connection refused
unit-opensearch-1: 01:44:24 ERROR unit.opensearch/1.juju-log node-lock-fallback:1: Cannot connect to the OpenSearch server...
unit-opensearch-1: 01:44:24 ERROR unit.opensearch/1.juju-log node-lock-fallback:1: [Errno 111] Connection refused
unit-opensearch-1: 01:44:24 ERROR unit.opensearch/1.juju-log node-lock-fallback:1: [Errno 111] Connection refused
unit-opensearch-1: 01:44:25 ERROR unit.opensearch/1.juju-log certificates:3: [Errno 111] Connection refused
unit-opensearch-1: 01:44:25 ERROR unit.opensearch/1.juju-log certificates:3: Cannot connect to the OpenSearch server...
unit-opensearch-1: 01:44:25 ERROR unit.opensearch/1.juju-log certificates:3: [Errno 111] Connection refused
unit-opensearch-1: 01:44:26 ERROR unit.opensearch/1.juju-log [Errno 111] Connection refused
unit-opensearch-1: 01:44:26 ERROR unit.opensearch/1.juju-log Cannot connect to the OpenSearch server...
unit-opensearch-1: 01:44:26 ERROR unit.opensearch/1.juju-log [Errno 111] Connection refused
unit-opensearch-1: 01:44:27 ERROR unit.opensearch/1.juju-log [Errno 111] Connection refused
unit-opensearch-1: 01:44:27 ERROR unit.opensearch/1.juju-log Cannot connect to the OpenSearch server...
unit-opensearch-1: 01:44:27 ERROR unit.opensearch/1.juju-log [Errno 111] Connection refused
unit-opensearch-1: 01:44:27 ERROR unit.opensearch/1.juju-log [Errno 111] Connection refused
unit-opensearch-0: 01:44:28 ERROR unit.opensearch/0.juju-log opensearch-peers:0: [Errno 111] Connection refused
unit-opensearch-0: 01:44:28 ERROR unit.opensearch/0.juju-log opensearch-peers:0: [Errno 111] Connection refused
unit-opensearch-1: 01:44:28 ERROR unit.opensearch/1.juju-log opensearch-peers:0: [Errno 111] Connection refused
unit-opensearch-1: 01:44:28 ERROR unit.opensearch/1.juju-log opensearch-peers:0: Cannot connect to the OpenSearch server...
unit-opensearch-1: 01:44:28 ERROR unit.opensearch/1.juju-log opensearch-peers:0: [Errno 111] Connection refused
unit-opensearch-0: 01:44:28 ERROR unit.opensearch/0.juju-log opensearch-peers:0: [Errno 111] Connection refused
unit-opensearch-0: 01:44:28 ERROR unit.opensearch/0.juju-log opensearch-peers:0: [Errno 111] Connection refused
unit-opensearch-0: 01:44:28 ERROR unit.opensearch/0.juju-log opensearch-peers:0: [Errno 111] Connection refused
unit-opensearch-1: 01:44:28 ERROR unit.opensearch/1.juju-log opensearch-peers:0: [Errno 111] Connection refused
unit-opensearch-1: 01:44:28 ERROR unit.opensearch/1.juju-log opensearch-peers:0: [Errno 111] Connection refused
unit-opensearch-1: 01:44:28 ERROR unit.opensearch/1.juju-log opensearch-peers:0: [Errno 111] Connection refused
unit-opensearch-1: 01:44:28 ERROR unit.opensearch/1.juju-log opensearch-peers:0: [Errno 111] Connection refused
unit-opensearch-1: 01:44:28 ERROR unit.opensearch/1.juju-log opensearch-peers:0: [Errno 111] Connection refused
unit-opensearch-1: 01:44:28 ERROR unit.opensearch/1.juju-log opensearch-peers:0: [Errno 111] Connection refused
unit-opensearch-1: 01:44:29 ERROR unit.opensearch/1.juju-log opensearch-peers:0: [Errno 111] Connection refused
unit-opensearch-1: 01:44:29 ERROR unit.opensearch/1.juju-log opensearch-peers:0: Cannot connect to the OpenSearch server...
unit-opensearch-1: 01:44:29 ERROR unit.opensearch/1.juju-log opensearch-peers:0: [Errno 111] Connection refused
unit-opensearch-1: 01:44:32 ERROR unit.opensearch/1.juju-log opensearch-peers:0: [Errno 111] Connection refused
unit-opensearch-1: 01:44:35 ERROR unit.opensearch/1.juju-log opensearch-peers:0: [Errno 111] Connection refused
unit-opensearch-1: 01:44:38 ERROR unit.opensearch/1.juju-log opensearch-peers:0: [Errno 111] Connection refused
machine-3: 01:47:25 ERROR juju.worker.dependency "kvm-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-3: 01:47:25 ERROR juju.worker.dependency "lxd-container-provisioner" manifold worker returned unexpected error: container types not yet available
unit-second-opensearch-0: 01:47:25 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
unit-second-opensearch-0: 01:48:41 ERROR unit.second-opensearch/0.juju-log [Errno 111] Connection refused
unit-second-opensearch-0: 01:48:42 ERROR unit.second-opensearch/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-second-opensearch-0: 01:48:42 ERROR unit.second-opensearch/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-second-opensearch-0: 01:48:42 ERROR unit.second-opensearch/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-second-opensearch-0: 01:48:42 ERROR unit.second-opensearch/0.juju-log opensearch-peers:4: [Errno 111] Connection refused
unit-second-opensearch-0: 01:48:43 ERROR unit.second-opensearch/0.juju-log certificates:7: [Errno 111] Connection refused
unit-second-opensearch-0: 01:48:44 ERROR unit.second-opensearch/0.juju-log upgrade-version-a:6: [Errno 111] Connection refused
unit-second-opensearch-0: 01:48:45 ERROR unit.second-opensearch/0.juju-log certificates:7: [Errno 111] Connection refused
unit-second-opensearch-0: 01:48:46 ERROR unit.second-opensearch/0.juju-log certificates:7: Uncaught exception while in charm code:
Traceback (most recent call last):
  File "/var/lib/juju/agents/unit-second-opensearch-0/charm/./src/charm.py", line 213, in <module>
    main(OpenSearchOperatorCharm)
  File "/var/lib/juju/agents/unit-second-opensearch-0/charm/venv/ops/main.py", line 551, in main
    manager.run()
  File "/var/lib/juju/agents/unit-second-opensearch-0/charm/venv/ops/main.py", line 530, in run
    self._emit()
  File "/var/lib/juju/agents/unit-second-opensearch-0/charm/venv/ops/main.py", line 519, in _emit
    _emit_charm_event(self.charm, self.dispatcher.event_name)
  File "/var/lib/juju/agents/unit-second-opensearch-0/charm/venv/ops/main.py", line 147, in _emit_charm_event
    event_to_emit.emit(*args, **kwargs)
  File "/var/lib/juju/agents/unit-second-opensearch-0/charm/venv/ops/framework.py", line 348, in emit
    framework._emit(event)
  File "/var/lib/juju/agents/unit-second-opensearch-0/charm/venv/ops/framework.py", line 860, in _emit
    self._reemit(event_path)
  File "/var/lib/juju/agents/unit-second-opensearch-0/charm/venv/ops/framework.py", line 950, in _reemit
    custom_handler(event)
  File "/var/lib/juju/agents/unit-second-opensearch-0/charm/lib/charms/tls_certificates_interface/v3/tls_certificates.py", line 1926, in _on_relation_changed
    self.on.certificate_available.emit(
  File "/var/lib/juju/agents/unit-second-opensearch-0/charm/venv/ops/framework.py", line 348, in emit
    framework._emit(event)
  File "/var/lib/juju/agents/unit-second-opensearch-0/charm/venv/ops/framework.py", line 860, in _emit
    self._reemit(event_path)
  File "/var/lib/juju/agents/unit-second-opensearch-0/charm/venv/ops/framework.py", line 950, in _reemit
    custom_handler(event)
  File "/var/lib/juju/agents/unit-second-opensearch-0/charm/lib/charms/opensearch/v0/opensearch_tls.py", line 229, in _on_certificate_available
    self.charm.on_tls_conf_set(event, scope, cert_type, renewal)
  File "/var/lib/juju/agents/unit-second-opensearch-0/charm/lib/charms/opensearch/v0/opensearch_base_charm.py", line 718, in on_tls_conf_set
    self.opensearch_config.set_node_tls_conf(
  File "/var/lib/juju/agents/unit-second-opensearch-0/charm/lib/charms/opensearch/v0/opensearch_config.py", line 86, in set_node_tls_conf
    self._opensearch.config.put(
  File "/var/lib/juju/agents/unit-second-opensearch-0/charm/lib/charms/opensearch/v0/helper_conf_setter.py", line 200, in put
    data = self.load(config_file)
  File "/var/lib/juju/agents/unit-second-opensearch-0/charm/lib/charms/opensearch/v0/helper_conf_setter.py", line 184, in load
    del data[random_id]
  File "/var/lib/juju/agents/unit-second-opensearch-0/charm/venv/ruamel/yaml/comments.py", line 916, in __delitem__
    ordereddict.__delitem__(self, key)
KeyError: '22331927e01443539947743039215629'
unit-second-opensearch-0: 01:48:47 ERROR juju.worker.uniter.operation hook "certificates-relation-changed" (via hook dispatching script: dispatch) failed: exit status 1
unit-second-opensearch-0: 01:48:52 ERROR unit.second-opensearch/0.juju-log certificates:7: [Errno 111] Connection refused
unit-second-opensearch-0: 01:48:53 ERROR unit.second-opensearch/0.juju-log [Errno 111] Connection refused
unit-second-opensearch-0: 01:48:54 ERROR unit.second-opensearch/0.juju-log [Errno 111] Connection refused
unit-second-opensearch-0: 01:48:55 ERROR unit.second-opensearch/0.juju-log [Errno 111] Connection refused
unit-second-opensearch-0: 01:48:56 ERROR unit.second-opensearch/0.juju-log certificates:7: [Errno 111] Connection refused
unit-second-opensearch-0: 01:48:56 ERROR unit.second-opensearch/0.juju-log certificates:7: [Errno 111] Connection refused
unit-second-opensearch-0: 01:48:56 ERROR unit.second-opensearch/0.juju-log certificates:7: Cannot connect to the OpenSearch server...
unit-second-opensearch-0: 01:48:56 ERROR unit.second-opensearch/0.juju-log certificates:7: [Errno 111] Connection refused
unit-second-opensearch-0: 01:48:57 ERROR unit.second-opensearch/0.juju-log certificates:7: [Errno 111] Connection refused
unit-second-opensearch-0: 01:48:57 ERROR unit.second-opensearch/0.juju-log certificates:7: Cannot connect to the OpenSearch server...
unit-second-opensearch-0: 01:48:57 ERROR unit.second-opensearch/0.juju-log certificates:7: [Errno 111] Connection refused
unit-second-opensearch-0: 01:48:57 ERROR unit.second-opensearch/0.juju-log certificates:7: Cannot connect to the OpenSearch server...
unit-second-opensearch-0: 01:49:00 ERROR unit.second-opensearch/0.juju-log certificates:7: [Errno 111] Connection refused
unit-second-opensearch-0: 01:49:00 ERROR unit.second-opensearch/0.juju-log certificates:7: Cannot connect to the OpenSearch server...
unit-second-opensearch-0: 01:49:03 ERROR unit.second-opensearch/0.juju-log certificates:7: [Errno 111] Connection refused
unit-second-opensearch-0: 01:49:03 ERROR unit.second-opensearch/0.juju-log certificates:7: Cannot connect to the OpenSearch server...
unit-second-opensearch-0: 01:49:06 ERROR unit.second-opensearch/0.juju-log certificates:7: [Errno 111] Connection refused
unit-second-opensearch-0: 01:49:06 ERROR unit.second-opensearch/0.juju-log certificates:7: Cannot connect to the OpenSearch server...
unit-second-opensearch-0: 01:50:59 ERROR juju.worker.uniter resolver loop error: preparing operation "run relation-departed (7; unit: self-signed-certificates/0, departee: second-opensearch/0) hook" for second-opensearch/0: secret "secret:crf5812qci22d0lr38m0" not found (not found)
unit-second-opensearch-0: 01:50:59 ERROR juju.worker.dependency "uniter" manifold worker returned unexpected error: preparing operation "run relation-departed (7; unit: self-signed-certificates/0, departee: second-opensearch/0) hook" for second-opensearch/0: secret "secret:crf5812qci22d0lr38m0" not found (not found)
unit-second-opensearch-0: 01:51:08 ERROR unit.second-opensearch/0.juju-log [Errno 111] Connection refused
unit-second-opensearch-0: 01:51:08 ERROR unit.second-opensearch/0.juju-log Cannot connect to the OpenSearch server...
unit-second-opensearch-0: 01:51:08 ERROR unit.second-opensearch/0.juju-log [Errno 111] Connection refused
unit-second-opensearch-0: 01:51:10 ERROR juju.worker.dependency "uniter" manifold worker returned unexpected error: relation scope id "e9c3e25d-60a0-4cf3-810d-eb09d30cd968:r#6#second-opensearch": settings e9c3e25d-60a0-4cf3-810d-eb09d30cd968:r#6#second-opensearch not found (not found)

[32mINFO    [0m pytest_operator.plugin:plugin.py:947 Forgetting model main...